tests/ExceptionTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Razorpay\Tests;
+
+use Razorpay\Api\Request;
+use Razorpay\Api\Errors;
+
+class ExceptionTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+    }
+    
+    /**
+     * Create an order from json payload
+     */
+    public function testCreateJsonOrderException()
+    {
+        $payload = $this->payload();
+
+        $attribute = json_encode($payload);
+        try
+        {
+         $data = $this->api->order->create($attribute);
+
+         $this->assertTrue(is_array($data->toArray()));
+
+        }catch(Error $e){
+            throw new InvalidArgumentException($e);
+        }
+    }
+
+    /**
+     * Create an order from set header application/json
+     */
+    public function testCreateOrderSetHeaderException()
+    {
+        $attribute = $this->payload();
+
+        try
+        {
+         $this->api->setHeader('content-type', 'application/json');
+
+         $data = $this->api->order->create($attribute);
+
+         $this->assertTrue(is_array($data->toArray()));
+        
+        }catch(Error $e){
+            throw new InvalidArgumentException($e);
+        }
+    }
+
+    /**
+     * Create an order
+     */
+    public function testCreateOrderSuccess()
+    {
+        $attribute = $this->payload();
+        try
+        {   
+         $data = $this->api->order->create($attribute);
+
+         $this->assertTrue(is_array($data->toArray()));
+
+        }catch(Error $e){
+            throw new InvalidArgumentException($e);
+        }
+    }
+    
+    private function payload(){
+        $date = new \DateTime();
+        $receiptId = $date->getTimestamp();
+        return array(
```

Comment:
@ankitdas13 , please use `[]` for array and do the proper formatting.

Reply:
changed `array()` method to `[]`